scripts/globals/spells/bluemagic/magic_fruit.lua

```diff
@@ -40,7 +40,7 @@ function onSpellCast(caster,target,spell)
 
     final = final + (final * (target:getMod(dsp.mod.CURE_POTENCY_RCVD)/100))
 
-    if (target:getAllegiance() == caster:getAllegiance() and (target:getObjType() == dsp.objType.PC or target:getObjType() == dsp.objType.MOB)) then
+    if (target:getAllegiance() == caster:getAllegiance() and (target:getObjType() == dsp.objType.PC or target:getObjType() == dsp.objType.MOB or target:getObjType() == dsp.objType.TRUST)) then
```

Comment:
These checks for trust object type might be problematic later for the likes of Sakura, Brygid, etc (sphere trusts), but those are a long ways off, so this is probably fine for now. Just noting as something to keep in mind for the future, in case we want to consider some "trust is targetable" flags / functions.

Teodor, being undead, will also require some special handling to make the cures hit for 0.